ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/ProtocolScheduleTest.java

```diff
@@ -32,6 +38,31 @@
 public class ProtocolScheduleTest {
```

Comment:
nit: rename this test to `MutableProtocolScheduleTest` for consistency?

Reply:
Agree